lua/obsidian/command.lua

```diff
@@ -168,13 +168,73 @@ command.search = function(client, data)
   end
 end
 
+--- Follow link under cursor.
+---
+---@param client obsidian.Client
+command.follow = function(client, _)
+  local scan = require "plenary.scandir"
+
+  local open, close = util.cursor_on_markdown_link()
+  local current_line = vim.api.nvim_get_current_line()
+
+  if open == nil or close == nil then
+    return nil
+  end
+
+  local note_name = current_line:sub(open + 2, close - 1)
+  local path = client.dir
+  
+  if not note_name:match("%.md") then
+    note_name = note_name .. ".md"
+  end
+  
+  if note_name:match("|[^%]]*") then
+    note_name = note_name:sub(1, note_name:find('|'))
+  end
+
+  local notes = {}
+  local count = 1
+
+  if not note_name:match("/") then
+    scan.scan_dir(vim.fs.normalize(tostring(client.dir)), {
+      hidden = false,
+      add_dirs = false,
+      respect_gitignore = true,
+      search_pattern = ".*%.md",
+      on_insert = function(entry)
+        Note.from_file(entry, client.dir)
+        local ok, note = pcall(Note.from_file, entry, client.dir)
+
+        if ok and note:fname() == note_name then
+          notes[count] = entry
+          count = count + 1
+        end
+      end,
+    })
+  end
+
+
+  if table.getn(notes) < 1 then
```

Comment:
Since `notes` is an array you can do
```suggestion
  if #notes < 1 then
```